rootfs/deploy.py

```diff
@@ -0,0 +1,48 @@
+import docker
+import os
+import tarfile
+import os.path
+import urllib
```

Comment:
I'm okay with using requests to read the body. Thought that might complicate the code .